lib/commons/models/hive_objects/insight_user_object.dart

```diff
@@ -0,0 +1,23 @@
+import 'package:hive/hive.dart';
+
+part 'insight_user_object.g.dart';
+
+@HiveType(typeId: 0)
+class InsightUser {
+  @HiveField(0)
+  late String? accessToken;
+
+  @HiveField(1)
+  late String? token;
+
+  @HiveField(2)
+  late String? idToken;
+
+  @HiveField(3)
+  late String? secret;
+
+  @HiveField(4)
+  late String? refreshToken;
+
+
```

Comment:
Add a toString() method to quickly show the object. `idToken` is perhaps a good parameter to display. Further discussion might be needed(?)
```dart
  @override
  String toString() {
    return 'InsightUser{ idToken: $idToken }';
  }
```